wine_worker_container/buildContainer.sh

```diff
@@ -0,0 +1 @@
+docker build -t worker .
```

Comment:
that will probably be included in the `deploy.sh` script in some way

Reply:
na ten moment to zostawić póki jeszcze nie jest częścią prod aplikacji?